supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-data-grid.adoc

```diff
@@ -0,0 +1,113 @@
+[[red-hat-data-grid-conventions]]
+
+For documentation questions, contact jdg-docs@redhat.com.
+
+[discrete]
+[[cache-manager]]
+==== image:images/yes.png[yes] Cache Manager (noun)
+*Description*: Interface for creating caches and managing cache lifecycles. Always spell as two words with capital letters when referring to the abstract notion of a Cache Manager. When referring to specific interfaces, use `CacheManager`, `EmbeddedCacheManager`, or `RemoteCacheManager`, marked up appropriately.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[cross-site-replication]]
+==== image:images/yes.png[yes] cross-site replication (noun)
+*Description*: A configuration that allows Data Grid clusters to form a global view and back up data across geographically disperse locations. Multiple clusters running in different data centers replicate data between each other to ensure business continuity in the event of an outage and to present a single, unified caching service to applications.
+
+*Use it*: yes
+
+*Incorrect forms*: xsite
+
+*See also*:
+
+[discrete]
+[[data-grid]]
+==== image:images/yes.png[yes] Data Grid (noun)
+*Description*: The short product name for Red Hat Data Grid. Use "Red Hat Data Grid" in the first instance and "Data Grid" in all subsequent instances.
+
+*Use it*: yes
+
+*Incorrect forms*: JBoss Data Grid
+
+*See also*: xref:red-hat-data-grid[Red Hat Data Grid]
+
+[discrete]
+[[data-grid-console]]
+==== image:images/yes.png[yes] Data Grid Console (noun)
+*Description*: Data Grid Console provides a web interface for creating and managing caches.
+
+*Use it*: yes
+
+*Incorrect forms*: JBoss Data Grid Administration Console, Data Grid console
+
+*See also*:
+
+[discrete]
+[[data-grid-server]]
+==== image:images/yes.png[yes] Data Grid Server (noun)
+*Description*: Data Grid Server is a standalone server which exposes any number of caches to clients over a variety of protocols, including Hot Rod, Memcached and REST. Always capitalize when referring to a Data Grid Server instance.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[hot-rod]]
+==== image:images/yes.png[yes] Hot Rod (adjective)
+*Description*: Hot Rod is a binary TCP client-server protocol used in Red Hat Data Grid. Java, C#, C++, Node.js clients, as well as clients written in other programming languages, and clients can access data that resides in remote caches on Data Grid Server clusters via the Hot Rod endpoint. Write as two words and capitalize the first letter of each word.
```

Comment:
I'm getting a smidge tripped up parsing this sentence. It seems like there's something missing. Could you try to simplify it a bit? (it might be just my brain today though!)

`Java, C#, C++, Node.js clients, as well as clients written in other programming languages, and clients can access data that resides in remote caches on Data Grid Server clusters via the Hot Rod endpoint.